.gitignore

```diff
@@ -1,7 +1,8 @@
 .kitchen/
 credentials.json
-test/**/terraform.tfvars
-test/**/.terraform
-test/**/terraform.tfstate.d
-test/**/ssh/key
+examples/**/terraform.tfvars
+examples/**/.terraform
+examples/**/terraform.tfstate.d
+examples/**/ssh/key
 *.pyc
+test/fixtures/terraform.tfvars
```

Comment:
I'd suggest locating this in the examples directory instead.

Reply:
@morgante Might I ask why? The variables file seems more relevant as a test-related fixture to me.

Reply:
You also need to fill in the variables to run the examples too.